hosters/hetzner-dedicated/hetzner-dedicated-wipe-and-install-nixos.sh

```diff
@@ -147,17 +233,15 @@ mount /dev/disk/by-label/root /mnt
 
 # Installing nix
 
-# Installing nix requires `sudo`; the Hetzner rescue mode doesn't have it.
-apt-get install -y sudo
-
 # Allow installing nix as root, see
 #   https://github.com/NixOS/nix/issues/936#issuecomment-475795730
 mkdir -p /etc/nix
 echo "build-users-group =" > /etc/nix/nix.conf
```

Comment:
```suggestion
echo "build-users-group =" > /etc/nix/nix.conf
echo "experimental-features = nix-command" > /etc/nix/nix.conf
```

Reply:
Note: this should be `>>` in line 240 (to retain the `build-users-group`).